.github/workflows/rebuild_c_files.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+
+name: Rebuild .c files with latest Cython
+jobs:
+  rebuild_c_files:
+    name: Rebuild .c files with latest Cython
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install dependencies
+        run: |
+          pip install cython
+          echo "cython_version=$(pip freeze | grep -i cython | cut -d'=' -f3)" >> $GITHUB_ENV
+      - name: Rebuild .c files with latest Cython
+        uses: technote-space/create-pr-action@v2
```

Comment:
This action (https://github.com/marketplace/actions/create-pr-action) executes an arbitrary command, commits the diff, and opens it as a PR. When there's no diff, it doesn't create a PR.

So most times this runs it won't create a PR. It'll only create a PR if deleting built files and then running `python setup.py build_ext --inplace` (below) results in a diff. That only happens when there's a new Cython version. The diffs look something like https://github.com/dimagi/jsonobject/pull/187/files.

Reply:
I'll leave a quick comment here to that effect

Reply:
I guess it's not possible to create a new PR if there is no diff, and hence nothing to commit. Thanks for clarifying.

Reply:
If a new version is published, workflow runs -> diff -> PR is created, and that PR is not merged before the workflow runs again it will create a second PR having the same contents as the first (assuming Cython has not changed). These PRs will continue to pile up, with a new one created each time the workflow runs, until someone merges one of them. Is that correct?

Reply:
That's a possibility; I haven't used this workflow before and I didn't test that. If it's smart it'll notice it already made the exact same PR before, but yeah if it's not it'll just keep making more. That would be an excellent reason to do monthly rather than daily!